### DISCUSSION TITLE
support more attributes in html tables

### DISCUSSION THREAD

**Discussion by cjhabs** [GitbookIO/community#633] [Feature Requests]
Hi
Table presentation is very limited currently.
The only "override" I could find that works is the "width" attribute for the columns.

I would like to be able to define a border for the table:
`<table border="1">`
This could be done using CSS also if that was supported.

I would also like to span columns:
`<tr><td colspan="3">This spans a 3 column table</td></tr>`

Being able to apply custom CSS to format the table header and stripe alternate rows for example, would be ideal.

Hopefully others have a need for this also.

Thanks

Chris

**Comment by JaniceManwiller** [comment]
Agree that the table formatting is very limited.

Other options that would be nice to have:

- Text alignment. Currently displays vertical center but really need to be able to set it to vertical top for readability.
- Add blocks within a cell. When I hit Enter, should start a new paragraph within the cell. Would also want to add small images and short lists.
